rotkehlchen/accounting/structures/base.py

```diff
@@ -289,10 +289,10 @@ def __str__(self) -> str:
     def get_timestamp_in_sec(self) -> Timestamp:
         return ts_ms_to_sec(self.timestamp)
 
-    def get_type_identifier(self) -> str:
+    def get_type_identifier(self, include_counterparty: bool = True) -> str:
         """A unique type identifier for known event types"""
```

Comment:
Add a docstring to describe what the presence of `include_counterparty` introduces.